test/metabase/driver/sql/test_util/unique_prefix.clj

```diff
@@ -40,6 +41,8 @@
    (unique-prefix* (utc-date)))
   ([local-date]
    {:pre [(instance? java.time.LocalDate local-date)]}
+   ;; app DB has to be initialized to get settings
+   (initialize/initialize-if-needed! :db)
```

Comment:
most of the little changes like this are to fix tests that fail if they're run on their own without other stuff getting run first